test/e2e/ipsec_test.go

```diff
@@ -49,24 +48,32 @@ func TestIPSec(t *testing.T) {
 	defer data.redeployAntrea(t, deployAntreaDefault)
 
 	t.Run("testIPSecPSKAuth", func(t *testing.T) {
-		cc := func(config *controllerconfig.ControllerConfig) {
-		}
-		ac := func(config *agentconfig.AgentConfig) {
-			config.IPsec.AuthenticationMode = "psk"
-		}
-		if err := data.mutateAntreaConfigMap(cc, ac, true, true); err != nil {
```

Comment:
do you know why we were restarting the controller before? was it a mistake?

Reply:
It should be a mistake. Previously the following tests will change the feature gates and AuthenticationMode to test cert based auth. But for PSK auth the controller should not be restarted.